### DISCUSSION TITLE
Slingbox Remote Bricking prevention

### DISCUSSION THREAD

**Discussion by eskwadrat** [GerryDazoo/Slinger#102] [General]
Anyone thought about how to prevent our Slinboxes from being accessed remotely by EchoStar (owner of Slingmedia) on or after the day of sling servers shutdown (Nov 9th, 2022) and possibly disabling Slinboxes (bricking)? It is not unheard of and such practice happened already few times with other products.

Simply, what would be the best way to somehow isolate Slingbox from accessing internet and only serve Slinger server?

**Comment by GerryDazoo** [comment]
This was covered in another thread but I'll reiterate:
I think you're being overly paranoid. But if you must, block all incoming and outgoing packets to your slingbox in your router.
